assets/airports/lszh.json

```diff
@@ -0,0 +1,1812 @@
+        "ZH363": ["N47.865092", "E8.528153"],
+        "ZH365": ["N47.724570", "E8.554345"],
+        "ZH367": ["N47.543397", "E8.679225"],
+        "ZH368": ["N47.452758", "E8.741347"],
+        "ZH369": ["N47.362081", "E8.803256"],
+        "ZH370": ["N47.271408", "E8.864928"],
+        "ZH502": ["N47.465217", "E8.766339"],
+        "ZH503": ["N47.575000", "E8.709722"],
+        "ZH504": ["N47.456389", "E8.896945"],
+        "ZH506": ["N47.507222", "E8.780833"],
+        "ZH525": ["N47.440100", "E9.011075"],
+        "ZH526": ["N47.259269", "E8.620970"],
+        "ZH551": ["N47.302222", "E8.166667"],
+        "ZH552": ["N47.428889", "E8.391667"],
+        "ZH553": ["N47.422222", "E8.356111"],
+        "ZH554": ["N47.355086", "E8.248747"],
+        "ZH555": ["N47.346889", "E8.261267"],
+        "ZH556": ["N47.338333", "E8.384722"],
+        "ZH557": ["N47.313056", "E8.403611"],
+        "ZH558": ["N47.318056", "E8.144722"],
+        "ZH559": ["N47.517083", "E8.384667"],
+        "ZH570": ["N47.518000", "E8.505583"],
+        "ZH573": ["N47.534167", "E8.436667"],
+        "ZH578": ["N47.502695", "E8.459167"],
+        "ZH627": ["N47d22m20.7", "E8d37m13.7"],
+        "ZH628": ["N47d16m9.1", "E8d41m28.0"],
+        "ZH650": ["N47.362778", "E8.533333"],
+        "ZH676": ["N47.461111", "E8.504445"],
+        "ZH677": ["N47.577222", "E7.736945"],
+        "ZH701": ["N47.630833", "E8.667778"],
+        "ZH703": ["N47.485103", "E8.936397"],
+        "ZH713": ["N47.595306", "E8.337528"],
+        "ZH725": ["N47.253333", "E8.798056"],
+        "ZH726": ["N47.247222", "E8.624472"],
+        "ZH751": ["N47.556583", "E8.742889"],
+        "ZH775": ["N47.588333", "E8.128889"],
+        "ZUE"  : ["N47d35m31.00", "E8d49m3.00"]
+    },
+    "restricted": [
+        {
+            "name": "LS-R31",
+            "height": "8500ft",
+            "coordinates": [
+                ["N47d11m16", "E08d28m04"],
+                ["N47d10m09", "E08d30m00"],
+                ["N47d07m55", "E08d29m40"],
+                ["N47d02m15", "E08d23m11"],
+                ["N47d01m50", "E08d20m18"],
+                ["N47d03m54", "E08d19m19"],
+                ["N47d08m19", "E08d24m18"]
+            ]
+        }
+    ],
+    "runways": [
+        {
+            "name": ["10", "28"],
+            "end": [
+                ["N47.459000", "E8.537500", "1391ft"],
+                ["N47.456667", "E8.570500", "1416ft"]
+            ],
+            "ils": [false, true]
+        },
+        {
+            "name": ["14", "32"],
+            "end": [
+                ["N47.482167", "E8.536000", "1405ft"],
+                ["N47.461333", "E8.564500", "1402ft"]
+            ],
+            "ils": [true, false]
+        },
+        {
+            "name": ["16", "34"],
+            "end": [
+                ["N47.475667", "E8.536000", "1390ft"],
+                ["N47.449333", "E8.554167", "1385ft"]
+            ],
+            "ils": [true, true]
+        }
+    ],
+    "sids": {

```

Comment:
Missing 
```
WIL2C
WIL2Q
ZUE3F
ZUE1G
ZUE1M
WIL3A
ZUE2A
DEGES1R
DEGES1E
DEGES2H
DEGES2N
VEBIT1K
DEGES2B
VEBIT3B
```

These _can_ be added as a result of #809 being merged.

Reply:
Since WIL and ZUE aren't used I leave them out for now. I'm not going to add those missing DEGES and VEBIT routes since their usage depends on the current runway configuration, and in the 14 arrival runway - 28/16 departure runway, it doesn't matter which route is used for departures out of runway 32/34.

Reply:
I guess you're assuming the runway configuration will _always_ stay the same. If we do add varying winds and the runway configuration does change, it appears that it will matter which route is used for departure (from what you're implying)

Reply:
I'll just pick DEGES2H out of your list as an example.
DEGES2N is a SID for runway 32. Currently the airport file is including DEGES3L for runway 32. DEGES2N's route is going to the right after a few miles and DEGES3L to the left. Usually when runway 32 is used for departures, runway 28 is used for landings (f.e. during the German curfew period). Therefore DEGES2N isn't used since that would cause interference with inbound traffic for runway 28. Same is also applicable on runway 34. (In the east concept (landings from the east on runway 28) runway 34 is only used for aircraft which are too heavy for runway 32.) DEGES2N and DEGES2H (routes to the right) are only used in morning periods when lots of heavy aircraft are inbound from long haul destinations. Since runway 28 is too short for them and runway 14 can't be used due to the German curfew period all aircraft are using runway 34 for landings. Then DEGES2N and DEGES2H can be used since there's no interference with traffic landing on runways 28. Source: https://www.youtube.com/watch?v=z5W8CU0DjsM (unfortunately no subtitles) but there's also information about the operational concepts of LSZH in the charts.

As you see it's a bit more complicated here and if there would be a new feature with varying winds the user would also need to be able to change the spawnpatterns and change the SIDs. Since the runway configuration with landings on 28 and departures on 32 is way more likely than landings on 34 and departures on 32, I came to the conclusion that it's better to just leave out the mentioned SIDs and just focus on those for landings on runway 28. 
Also the configuration with landings on runway 34 is as I said only used in morning periods with few traffic it probably won't even come up here because the traffic volume always stays the same and with ~ 40acph this runway config can't be used.

I hope it's at least partly understandable (it's hard to describe it properly in English for me)

Reply:
Thankfully both @indianbhaji and I speak German 😄 We understand that the missing procedures are obsolete with the current configuration, and are very rarely used. But we're trying to go for the highest level of realism possible, so if the airport has a procedure (even if it's used by just one airplane every year), it should still be added to the sim. If the swiss aviation authority decided that the procedure was important enough to be given a name and a place on the charts, it deserves to be in the sim. Really, the only exceptions here are the procedures activated by NOTAM or lost comms procedures.

Reply:
Oh didn't know that 😄 
Actually they aren't only obsolete with the current configuration but I'll add them.
Still there's no use in adding the additional WIL and ZUE SIDs, there isn't even a spawnpattern for them included.

Reply:
@Marcel510 the above information would be very nice to have in the airport guide though 😉

Reply:
DEGES2B, VEBIT3B are temporary procedures activated by NOTAM,
the rest is added now